app/views/kudos/new.html.erb

```diff
@@ -25,7 +25,7 @@
   </p>
   <p>
     <%=f.label :company_value %><br/>
-    <%=f.collection_select :company_value_id, CompanyValue.all, :id , :title %>
+    <%=f.select(:company_value_id, [['Honesty', 1], ['Ownership', 2],['Passion', 3], ['Accountability', 4]]) %>
```

Comment:
What would happen if let's say record with title "Ownership" wouldn't have id 2?

Reply:
Gludek means that you should populate this select with CompanyValues from the database because right now you have a hardcoded list, but if any admin of the application adds new company values to db, you want to automatically appear here

https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/select

Reply:
"On the new kudo form there is a select field with the following values: `Honesty`, `Ownership`, `Accountability`, `Passion`" as taken from the acceptance criteria. It would be a lot easier to me to just type:'CompanyValue.all, but as I understood the criteria it was meant to be hardcoded...